dotcom-rendering/src/web/components/TweetBlockComponent.importable.tsx

```diff
@@ -0,0 +1,79 @@
+import { css } from '@emotion/react';
+import { border, body } from '@guardian/source-foundations';
+import { useEffect } from 'react';
+import { unescapeData } from '../../lib/escapeData';
+
+type Props = {
+	element: TweetBlockElement;
+};
+
+// fallback styling for when JS is disabled
+const noJSStyling = css`
+	.nojs-tweet:not(.nojs-tweet-rendered) {
+		border: 1px solid ${border.secondary};
+		border-radius: 4px;
+		padding: 20px;
+		width: 100%;
+		margin-bottom: 16px;
+		${body.small()};
+	}
+
+	.nojs-tweet iframe {
+		/* Unfortunately due to how Twitter embeds work setting !important is the only way to overwrite tweet CSS */
+		/* stylelint-disable-next-line declaration-no-important */
+		width: 100% !important;
+	}
+
+	.nojs-tweet p {
+		padding-bottom: 10px;
+	}
+
+	a {
+		/* stylelint-disable-next-line color-no-hex */
```

Comment:
? Is this a twitter colour?

Reply:
No idea 🤷  - this is isn't my code, I renamed the file so it got green'd

Reply:
This is somewhat odd… usually git managed to see this as a rename :(